kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/update-domain.sh

```diff
@@ -209,7 +210,10 @@ updateDomainHome() {
   # a pod which will run a script that creates a domain. The script then will extract the domain
   # resource using  WDT's extractDomainResource tool. So, the following code loops until the
   # domain resource is created by exec'ing into the pod to look for the presence of domainCreate.yaml file.
-  POD_NAME=`kubectl get pods -n ${namespace} | grep ${JOB_NAME} | awk ' { print $1; } '`
+  while [ -z "${POD_NAME}" ] || [ "$POD_NAME" = "$OLD_POD_NAME" ]; do
+    POD_NAME=`kubectl get pods -n ${namespace} | grep ${JOB_NAME} | awk ' { print $1; } '`
+    echo "Pod name is $POD_NAME"
```

Comment:
This loop has no timeout, where-as the next line of code waits a max of 600 seconds. So perhaps this loop should check if too much time has passed?  If you choose to address this, then note that you can get current number of seconds using the build-in "SECONDS" env var.

Reply:
hmm, I am not sure if we need to time out here. The next line is waiting for the pod to reach the ready state, which may not happen if there is an error. But this one is just to get the pod name that is created.

Reply:
added the timeout as suggested